src/AppLayout.vue

```diff
@@ -1,12 +1,12 @@
 <template>
 	<div class="ela-app-layout">
-		<nav class="ela-side-menu" @click="rightShown = false">
+		<nav class="ela-side-menu">
```

Comment:
This would cause clicking on navigation menu to not close the rightModal. Why is it removed?

Reply:
@hit12369 I guess rightShown is unused as rightShown was removed in https://github.com/smartprix/el-admin/commit/d40d5f6961e5d7ae93b2eaaeaa94dc40dc483093#diff-2d55731211200d778376557f0425945c